client/apps/Planning/PlanningListSubNav.tsx

```diff
@@ -22,14 +22,17 @@ interface IProps {
     currentInterval: 'DAY' | 'WEEK' | 'MONTH';
     sortOrder: SORT_ORDER;
     sortField: SORT_FIELD;
-
+    userList:Array<{label: string, onSelect(): void}>;
+    users: any;
     setStartFilter(value?: moment.Moment): void;
     jumpTo(interval: 'TODAY' | 'BACK' | 'FORWARD'): void;
     setJumpInterval(interval: 'DAY' | 'WEEK' | 'MONTH'): void;
+    activefilter?:string;
```

Comment:
We have an enum `FILTER_TYPE` in the `interfaces.ts` file, which can be used here. This would provide better type checking when comparing this value